src/main/server.ts

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env node
 
 import './ts-paths-bootstrap'
+
+import { AppInsights } from 'modules/app-insights'
+// App Insights needs to be enabled as early as possible as it monitors other libraries as well
+AppInsights.enable()
```

Comment:
what happens if you don't have the key set?

Reply:
Nothing, it just starts up normally.

Reply:
Or sec, there is a default for the key.

Reply:
It will fail if I remove the default.

Reply:
I guess I'll remove the default and set the dummy value for `development`, `dockertests` and `mocha`?